plugins/wasm_bpf/CMakeLists.txt

```diff
@@ -6,28 +6,40 @@
 # - ${LIBBPF_SOURCE_DIR}
 # - FetchContent
 
+option(WASMEDGE_PLUGIN_WASM_BPF_BUILD_LIBBPF_WITH_PKG_CONF "Build libbpf without pkg-config. Requires libelf and libz being placed at the proper place." YES)
```

Comment:
I am confused. The option said `WITH PKG-CONFIG`, however, the message said `WITHOUT PKG-CONFIG`.

Reply:
It's my fault. I forget to change the description